index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang= "en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portofolio</title>
+    <link rel="stylesheet" href="style.css">
+    </head>
+<body>
+<section class="Workssection1">
+    <header class="Header">
+        <a class="mylogo">MY logo</a>
+         <nav>
+            <a href="#"> </a>
+            <img href src="./image/logobuger.JPG">
+         </nav>
+    </header>
+     <div class="cardBack01">
+        <div class="mainiam">
+            <h2 > I'm Mihertab Glad to see you!</h2>
+        </div>
+        <div class="section_01">
+            <p > I’m a software developer! I can help you build a product , 
+                feature or website Look through some of my work and experience! If you like what 
+                you see and have a project you need coded, don’t hestiate to contact me. </p>
+        </div>
+        <div class = "social-section">
+                    <h4 class="connectmywork"> LET’S CONNECT</h4>
+                <ul class="social-icons-mywork">
+                    <li><a href="#"><img src="./image/Vectortweet.png" alt=""></a></li>
+                    <li><a href="#"><img src="./image/VectorM.jpg" alt=""></a></li>
+                    <li><a href="#"><img src="./image/Vector5.png" alt=""></a></li>
+                    <li><a href="#"><img src="./image/VectorGit.png" alt=""></a></li>
+                    <li><a href="#"><img src="./image/Linkedin.svg" alt=""></a></li>
+                </ul>
+        </div>
+        
+      </div>
```

Comment:
- [ ] Dear @DamenaM, this section should be a single section with a `section` tag, I see you surround all pages with the `section` tag, but this alone should be a section and the `header` should alone be a `header` section. both represents generic standalone sections of the document.
- [ ] Also, for the main title `I'm Mihertab Glad to see you!`, please use `h1` tag. This will be a great help when it comes to semantics.
- [ ] Another thing to improve here is the margins (top, bottom, left, right) defined in the design. The minimum mobile screen size your design should work well is 375px. And right now, it's overflowing a bit right now.

![Capture](https://user-images.githubusercontent.com/69077061/205220487-8cf746b5-9e79-4f76-9108-30ea5835fa81.PNG)